kate-website2.html

```diff
@@ -0,0 +1,165 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Kate Harrison</title>
+    <link rel="stylesheet" href="reset.css">
+    <link rel="stylesheet" href="kate-website2.css">
+    <a name="top"></a> <!-- https://www.computerhope.com/issues/ch001475.htm#top -->
+</head>
+<body>
+    <!-- https://www.w3schools.com/howto/howto_js_topnav.asp -->
+    <!-- https://forum.freecodecamp.org/t/how-to-link-navbar-buttons-to-different-parts-of-a-website/201748 -->
+       <div class="header">
```

Comment:
Look into semantic HTML, rather than using divs for each element.

```suggestion
       <header>
```